core/bridges/orm.go

```diff
@@ -179,26 +185,43 @@ func (o *orm) UpdateBridgeType(ctx context.Context, bt *BridgeType, btr *BridgeT
 	return err
 }
 
-func (o *orm) GetCachedResponse(ctx context.Context, dotId string, specId int32, maxElapsed time.Duration) (response []byte, err error) {
+func (o *orm) GetCachedResponse(ctx context.Context, dotId string, specId int32, maxElapsed time.Duration) ([]byte, error) {
+	// prefer to get latest value from cache
+	cachedResponse, inCache := o.bridgeLastValueCache.Load(responseKey(dotId, specId))
```

Comment:
How do we manage the case where the response is expired?

Reply:
I could store the time coming in and going out and return an error when `maxElapsed` is less than or equal to the stored time value. I started to do that, but left it out to keep the original query in-tact.

Reply:
Why not persist the value as a struct like `{val, time}` then you can check to see if its expired on lookup as well as run a background cleanup routine for pruning stale values.